test/dashdb.dml.test.js

```diff
@@ -0,0 +1,65 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: loopback-connector-dashdb
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+'use strict';
+
+/* eslint-env node, mocha */
+process.env.NODE_ENV = 'test';
+
+require('should');
+
+const Transaction = require('loopback-connector').Transaction;
```

Comment:
It doesn't seem like `Transaction` is being used?

Reply:
Yep, I'll remove it.